src/backend/core/migrations/0002_remove_user_email_user_admin_email_and_more.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 5.0.3 on 2024-03-23 21:21
+
+import django.db.models.deletion
+import uuid
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0001_initial'),
+    ]
```

Comment:
We had decided to have a single migration file until first release. OK to replace 0001 initial instead ?